src/Language/PureScript/Docs/Types.hs

```diff
@@ -352,7 +352,7 @@ parseVersion' str =
 
 asModule :: Parse PackageError Module
 asModule =
-  Module <$> key "name" (P.moduleNameFromString <$> asString)
+  Module <$> key "name" (P.moduleNameFromString . T.pack <$> asString)
```

Comment:
Oh allright, I didn't know that existed. There is bunch of other places like this, I'll go through them thanks!